yt/yt/server/master/security_server/security_manager.cpp

```diff
@@ -3791,6 +3801,11 @@ class TSecurityManager
             DoAddMember(UsersGroup_, SuperusersGroup_);
         }
 
+        // admins
+        if (EnsureBuiltinGroupInitialized(AdminsGroup_, AdminsGroupId_, AdminsGroupName)) {
```

Comment:
This change requires reign (https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/server/master/cell_master/serialize.h#L229) promotion, since it changes semantics of snapshot load.